test/suite/test_alter01.py

```diff
@@ -29,6 +29,12 @@
 import wttest
 from wtscenario import make_scenarios
 from helper_tiered import TieredConfigMixin, gen_tiered_storage_sources
+#from abstract_test_case import AbstractWiredTigerTestCase
```

Comment:
Dead code can be removed?

Reply:
Done. Yes, this was checked in by mistake. I will checked all test/suite/*py and removed from test_alter01.py and test_hs21.py.